templates/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```
@@ -25,7 +26,18 @@
                     "referrerDomain": { "type": "string" },
                     "dayOfWeek": { "type": "integer" },
                     "timeOfDay": { "type": "integer" },
-                    "pageCategory": { "type": "integer" },
+                    "pageCategory": {
+                        "type": "integer",
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "classifierResults": {
+                        "type": "object",
+                        "properties": {
+                            "additionalProperties": {
+                                "type": "integer"
+                            }
+                        }
+                    },
```

The `additionalProperties` field should be unnested one level based on the example. This schema generates a struct with an `additional_properties` field, instead of a key-value pair.

```diff
>                 "fields": [
>                   {
>                     "mode": "NULLABLE",
>                     "name": "additional_properties",
>                     "type": "INT64"
>                   }
>                 ],
>                 "mode": "NULLABLE",
>                 "name": "classifier_results",
>                 "type": "RECORD"
>               },
```

Try this instead:

```suggestion
                    "classifierResults": {
                        "type": "object",
                        "properties": {}
                        "additionalProperties": {
                            "type": "integer"
                        }
                    },
```